openapi_tester/validators.py

```diff
@@ -179,28 +181,34 @@ def validate_pattern(schema_section: dict[str, Any], data: str) -> str | None:
 def validate_multiple_of(
     schema_section: dict[str, Any], data: int | float
 ) -> str | None:
+    if not isinstance(data, (int, float)):
+        return None
     multiple = schema_section.get("multipleOf")
     if multiple and data % multiple != 0:
         return VALIDATE_MULTIPLE_OF_ERROR.format(data=data, multiple=multiple)
     return None
 
 
 def validate_maximum(schema_section: dict[str, Any], data: int | float) -> str | None:
+    if not isinstance(data, (int, float)):
+        return None
     maximum = schema_section.get("maximum")
     exclusive_maximum = schema_section.get("exclusiveMaximum")
-    if maximum and exclusive_maximum and data >= maximum:
+    if isinstance(maximum, (int, float)) and exclusive_maximum and data >= maximum:
```

Comment:
Changing this check becomes necessary as otherwise it wasn't working as expected when `maximum` or `minimum` were equal to `0`.